python/pytest.ini

```diff
@@ -0,0 +1,22 @@
+[pytest]
```

Comment:
Just leaving a drive-by comment... much of this information is already in `python/pyproject.toml`, and I think it should stay there and not be duplicated into `pytest.ini`.

@mroeschke just recently removed this project's `pytest.ini` in #5877, in favor of keeping as much tool configuration as possible in `pyproject.toml`.

Reply:
Makes sense, thanks! I have a very small preference for storing these types of things in `pyproject.toml` over `pytest.ini`, but more importantly... these configurations should only be in one place within any given repo. To avoid needing to have to think about how `pytest` resolves merging configuration from multiple sources.

Reply:
Makes total sense, it might have been me who screwed up with merging upstream. 🙂